crates/orchestrator/src/jobs/mod.rs

```diff
@@ -234,6 +235,10 @@ pub async fn process_job(id: Uuid, config: Arc<Config>) -> Result<(), JobError>
     let external_id = match AssertUnwindSafe(job_handler.process_job(config.clone(), &mut job)).catch_unwind().await {
         Ok(Ok(external_id)) => {
             tracing::debug!(job_id = ?id, "Successfully processed job");
+            // Add the time of processing to the metadata.
+            if job.job_type == JobType::ProofCreation {
```

Comment:
why only for proof creation? we can add for everything I think

Reply:
Alright, added.